css-view-transitions-2/Overview.bs

```diff
@@ -209,61 +209,91 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 # CSS rules # {#css-rules}
 
-## The <dfn id="at-auto-view-transition-rule">''@auto-view-transition''</dfn> rule ## {#auto-view-transition-rule}
+## The <dfn id="at-view-transition-rule">''@view-transition''</dfn> rule ## {#view-transition}
```

Comment:
I'm not sure how the `{#view-transition}` syntax is referenced, but keeping "-rule" might be useful to disambiguate from `::view-transition`

Reply:
Fixed